src/components/gauge/gauge.e2e.ts

```diff
@@ -0,0 +1,213 @@
+  //     await page.setContent(`
+  //       <gx-gauge type='circle' min-value='0' value='25' show-value='true' style-shadow='false'>
+  //         <gx-gauge-range color='rgba(116, 16, 216, 1)' amount='100' name='Violet'></gx-gauge-range>
+  //       </gx-gauge>
+  //     `);
+  //     await page.waitForChanges();
+  //     element = await page.find("gx-gauge");
+  //     await page.waitForChanges();
+  //     expect(await element.find("svg")).toBeTruthy();
+  //     expect(await element.find("span.marker")).toBeTruthy();
+  //   });
+
+  //   it("should set the range name in linear gauge", async () => {
+  //     await page.setContent(`
+  //       <gx-gauge type='line' min-value='0' value='25' show-value='true' style-shadow='false'>
+  //         <gx-gauge-range color='rgba(116, 16, 216, 1)' amount='100' name='Violet'></gx-gauge-range>
+  //       </gx-gauge>
+  //     `);
+  //     await page.waitForChanges();
+  //     element = await page.find("gx-gauge");
+  //     const rangeNameDiv = await element.find("span.rangeName");
+  //     await page.waitForChanges();
+  //     expect(await element.find(".gaugeContainerLine")).toBeTruthy();
+  //     expect(rangeNameDiv).toBeTruthy();
+  //     expect(rangeNameDiv.innerHTML).toEqual("Violet");
+  //     expect(await element.find(".minMaxDisplay")).toBeTruthy();
+  //   });
+  //   /////////////////// gx-range tests ///////////////////////
+  //   it("should correctly draw the range in linear gauge", async () => {
+  //     await page.setContent(
+  //       `<gx-gauge type='line' min-value='0' value='25' show-value='true' style-shadow='false' thickness='50'>
+  //         <gx-gauge-range amount='100' name='Violet' color="purple"></gx-gauge-range>
+  //       </gx-gauge>`
+  //     );
+  //     await page.waitForChanges();
+  //     element = await page.find("gx-gauge");
+  //     await page.waitForChanges();
+  //     const rangeContainer = await element.find("div.rangesContainer");
+  //     await page.waitForChanges();
+  //     const ranges = await rangeContainer.find("div.range");
+  //     await page.waitForChanges();
+  //     expect(rangeContainer).toBeTruthy();
+  //     expect(ranges).toBeTruthy();
+  //   });
+
+  //   it("should correctly draw the range in circular gauge", async () => {
+  //     await page.setContent(`
+  //       <gx-gauge type='circle' min-value='0' value='25' show-value='true' style-shadow='false' thickness='50'>
+  //         <gx-gauge-range color='rgba(116, 16, 216, 1)' amount='100' name='Violet'></gx-gauge-range>
+  //       </gx-gauge>"
+  //     `);
+  //     await page.waitForChanges();
+  //     element = await page.find("gx-gauge");
+  //     await page.waitForChanges();
+  //     const range = await element.find("svg circle");
+  //     await page.waitForChanges();
+  //     expect(await element.find("svg")).toBeTruthy();
+  //     expect(range.getAttribute("stroke")).toEqual("rgba(116, 16, 216, 1)");
+  //   });
+});
```

Comment:
What are the plans for this commented code?

Reply:
I will remove all comments at the last step!
For now, those comments help me to plan next test cases.

Reply:
I forgot to comment on the new fixes! 😅
I removed those comments in the last commit 4e26143